changelog.d/428.change.rst

```diff
@@ -0,0 +1,2 @@
+Fixes a bug where deserialized objects with ``cache_hash=True`` could
```

Comment:
- Please check out: https://github.com/python-attrs/attrs/blob/master/.github/CONTRIBUTING.rst#changelog

   Most notably: simple past + [semantic newlines](https://rhodesmill.org/brandon/2012/one-sentence-per-line/).
- Also this is specifically about `pickle` so that should be mentioned.
- This is a breaking change, see below for an explanation.

Reply:
Changelog entry updated